.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
```

Comment:
I'm not sure you'll be able to do this when we're signing code.

Reply:
It still can work in validate mode.